source/form.html

```diff
@@ -82,7 +82,7 @@ <h2 class="promo__title">Подбор программы</h2>
             <legend class="form__legend form__legend--contacts">Контактные данные (владельца кота)</legend>
             <div class="form__wrapper">
               <div class="form__content">
-                <label class="form__label" for="form-mail">E-mail:*</label>
+                <label class="form__label" for="form-mail">E-&nbspmail:*</label>
```

Comment:
```suggestion
                <label class="form__label" for="form-mail">E-&nbsp;mail:*</label>
```

точку с запятой забыла